File: app/src/main/java/org/mesonet/app/site/caching/FavoriteSiteRealmModel.java
package org.mesonet.app.site.caching;


import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class FavoriteSiteRealmModel extends RealmObject
{
    @PrimaryKey
    private String mStid;



    public String GetName()
    {
        return mStid;
    }


    public FavoriteSiteRealmModel(){}



    public FavoriteSiteRealmModel(String inStid)
    {
        mStid = inStid;
    }
}

File: app/src/main/java/org/mesonet/app/site/caching/FavoriteSiteRealmModel.kt
package org.mesonet.app.site.caching


import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class FavoriteSiteRealmModel : RealmObject {
    @PrimaryKey
    var mStid: String? = null


    fun GetName(): String? {
        return mStid
    }


    constructor() {}


    constructor(inStid: String) {
        mStid = inStid
    }
}
